src/profile-logic/call-tree.js

```diff
@@ -207,9 +207,13 @@ export class CallTree {
         ? _formatIntegerNumber
         : _formatDecimalNumber;
 
+      const formattedTotalTime = formatNumber(totalTime);
+      const formattedSelfTime = formatNumber(selfTime);
       displayData = {
-        totalTime: formatNumber(totalTime),
-        selfTime: selfTime === 0 ? '—' : formatNumber(selfTime),
+        totalTime: formattedTotalTime,
+        totalTimeWithUnit: formattedTotalTime + ' ms',
+        selfTime: selfTime === 0 ? '—' : formattedSelfTime,
+        selfTimeWithUnit: selfTime === 0 ? '—' : formattedSelfTime + ' ms',
```

Comment:
I'm a bit picky here: in all other places, we've put "ms" without the space before. I think we should do the same here.

Reply:
Actually I see we have a `formatMilliseconds` in `format-numbers.js`, we could use it here to make it easier to change (and maybe localize ?) later.

Reply:
Good catch! We should definitely use the same convention.
If we use `formatMilliseconds`, we'll lose the current number localization used in the flame graph tooltips until we localize that function. Is that ok?

Reply:
ah right, I thought this `formatNumber` was the same as the other one. Maybe we should localize it everywhere, but then it's for another PR and I'd rather not change it here. So let's not use `formatMilliseconds` for now, but please file an issue :)

Reply:
Sounds good!